README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# hw01
+
+Hello, my name is **Dinesh Das Gupta**. I am a Second Year in The College, studying Public Policy and Environmental Studies. I represent the Class of 2022 on College Council and sing with the a cappella group Voices in Your Head (Voices).  Voices released an album last year titled  *Begin Again*.  I am studying from Washington, DC, my hometown. I took a gap year before attending UChicago and during that time I worked for:
+    
+* The Institute of International Finance
+    + Here I analysed capital flows in Excel.
+* The Energy and Resources Institute
+    + I helped organize the World Sustainable Development Summit.
+* Goodwin for DC Council-at-Large
+    + I ran campaign operations for my local Ward.
+* Paulette Jordan for Governor of Idaho
+    + I completed tasks for the campaign leadership and Paulette.  
```

Comment:
Great use of bullet points! This section is a little hard to read due to it being staggered. I would suggest omitting the sub-items and using other Markdown syntax to make these lines a little more concise